rust/capture/src/utils.rs

```diff
@@ -58,7 +58,9 @@ pub fn replace_invalid_hex_escape_strings(
         // First, figure out if this byte is the start of an escape sequence,
         // and if it is, set the flag and move forward
         if bytes[i] == b'\\' {
+            println!("found possible escape sequence at {}", i);
```

Comment:
is this how we handle logging in rust, e.g. where we'd use structlog in python

or is it vestigial from figuring out?

Reply:
Gah, should have deleted. In rust we use the `error!()`, `warn!()` etc set of macros, from the `tracing` crate

Reply:
Thanks for catching